### DISCUSSION TITLE
reduce quickfilter width

### DISCUSSION THREAD

**Discussion by teafmartins** [serenity-is/Serenity#7109] [Help]
Hi, how can I reduce width of quickfilter?

Example:
![image](https://github.com/serenity-is/Serenity/assets/17210232/605a05df-0192-4be9-934a-0140b9b727cd)

Reduce to:

![image](https://github.com/serenity-is/Serenity/assets/17210232/a4649d99-51ac-478f-8c90-71da2fc99002)

Thanks

**Comment by VictorTomaili** [comment]
You can use css etc. `min-width: 75px; width: 75px;` set both of them.
